packages/@aws-cdk/aws-scheduler-alpha/README.md

```diff
@@ -138,6 +138,21 @@ new Schedule(this, 'Schedule', {
 });
 ```
 
+### Configuring a start and end time of the Schedule
+
+If you choose a recurring schedule, you can set the start and end time of the Schedule by specifying the startDate and endDate.
```

Comment:
```suggestion
If you choose a recurring schedule, you can set the start and end time of the Schedule by specifying the `startDate` and `endDate`.
```

Reply:
I would also prefer the props to be just named `start` and `end`, or `startTime`/`endTime`.

Reply:
In CFn, it is StartDate and EndDate, so I shortened it to start and end.